README.md

```diff
@@ -55,7 +55,6 @@ each test and suite fulfillment process to separate files. To use that configure
 log4j.rootLogger=INFO, TEST_FRAMEWORK_LOGGING_APPENDER
```

Comment:
is it breaking change in relation to Presto?

Reply:
I didn't notice any regression. I tested it locally.